### PR TITLE
feat(logs): add editor highlight for log file

### PR DESCRIPTION
**Pull request by oceanlvr**

### CODE REVIEW

**Comment by oceanlvr** [comment]
log page highlight style:
image

**Comment by oceanlvr** [comment]
there is still some problem under `dark` mode.
image

File: src/components/Editor.vue [hunk 1]
```diff
@@ -50,7 +51,32 @@ export default class Editor extends Vue {
...
+          [/\[ERROR\]/i, 'custom-error'],
+          [/\[FATAL\]/i, 'custom-fatal'],
+          [/\[[0-9][a-zA-Z.0-9:\-]+\]/i, 'custom-date'],
+        ],
+      },
+    })
+  }
+
   public initEditor(): void | boolean {
+    this.initCustomerLanguages()
```
Comment (0721Betty) [inline src/components/Editor.vue:79]: otherEditors may not need initCustomerLanguages

File: src/components/Editor.vue [hunk 2]
```diff
@@ -123,17 +149,22 @@ export default class Editor extends Vue {
...
-      return 'vs'
+    switch (this.theme) {
+      case 'dark':
+        return 'editor-dark'
+        break
+      case 'night':
+        return 'editor-night'
+        break
     }
+    return 'vs'
```
Comment (0721Betty) [inline src/components/Editor.vue:167]: use default case?

File: src/components/Editor.vue (hunk 1 above)
Comment (oceanlvr) [reply src/components/Editor.vue:80]: ok I will add some if statement.

File: src/components/Editor.vue (hunk 2 above)
Comment (oceanlvr) [reply src/components/Editor.vue:170]: yes, I just refactor the `if-else` statement to `switch-case`

**Comment by oceanlvr** [comment]
Current style in the `dark/night` mode.
image

**Review by ysfscream — APPROVED** [review]

**Comment by ysfscream** [comment]
> Current style in the `dark/night` mode.

It's looks better.

**Pull request merged**